pulp_container/app/tasks/sync_stages.py

```diff
@@ -330,7 +330,6 @@ def create_blob(self, man_dc, blob_data):
             url=blob_url,
             relative_path=digest,
             remote=self.remote,
-            extra_data={"headers": V2_ACCEPT_HEADERS},
```

Comment:
This line is not related to the fix, however sending manifest headers for the blob requests is wrong.